src/olympia/amo/templates/amo/fake_fxa_authorization.html

```diff
@@ -4,7 +4,12 @@
   <h2> {{ _('Register or Log in') }} </h2>
   <form id="fake_fxa_authorization" method="get" action="{{ url('auth:accounts.authenticate') }}">
     <h3> Don't panic! </h3>
-    <p> You're seeing this page instead of the regular authentication page because both <code>settings.DEBUG</code> and <code>settings.USE_FAKE_FXA_AUTH</code> are set to <code>True</code>.</p>
+    <p>
+      You're seeing this page instead of the regular authentication page because the
+      <code>FXA_CONFIG</code> being used has the client_id set to <code>.</code>.
```

Comment:
```suggestion
      You're seeing this page instead of the regular authentication page because the
      <code>FXA_CONFIG</code> being used has the client_id set to <code>.</code> and
      `<code>ENV</code> is set to <code>local</code>.
```